.github/workflows/win_python_matrix_test.yml

```diff
@@ -0,0 +1,85 @@
+name: Windows Pyomo Tests Workflow (Slim, Matrixed, Nightly)
+
+on: 
+  pull_request: 
+    branches:
+      - master
+
+jobs:
+  pyomo-tests:
+    name: Pyomo Test (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 5
+      matrix:
+        os: ['windows-latest']
+        python-version: ['2.7', '3.5', '3.6', '3.7', '3.8']
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }} with Miniconda
+      uses: goanpeca/setup-miniconda@v1
```

Comment:
Could you comment on this line? Is `goanpeca` a user or repo that we're pulling stuff from or is this some cryptic looking "standard boilerplate"?

Reply:
A comment has been added for this line.